babel/src/test/resources/sql/big-query.iq

```diff
@@ -1798,21 +1811,20 @@ SELECT
 #
 # Returns INT64
 
-!if (false) {
 SELECT
   DATETIME "2010-07-07 10:20:00" as first_datetime,
   DATETIME "2008-12-25 15:30:00" as second_datetime,
   DATETIME_DIFF(DATETIME "2010-07-07 10:20:00",
     DATETIME "2008-12-25 15:30:00", DAY) as difference;
-+----------------------------+------------------------+------------------------+
-| first_datetime             | second_datetime        | difference             |
-+----------------------------+------------------------+------------------------+
-| 2010-07-07T10:20:00        | 2008-12-25T15:30:00    | 559                    |
-+----------------------------+------------------------+------------------------+
++---------------------+---------------------+------------+
+| first_datetime      | second_datetime     | difference |
++---------------------+---------------------+------------+
+| 2010-07-07 10:20:00 | 2008-12-25 15:30:00 |        558 |
++---------------------+---------------------+------------+
+(1 row)
```

Comment:
difference has changed from 559 to 558. which is the correct value?

Add a comment to the test justifying the value.

Reply:
I ran this test for timestamp_diff on a fresh branch cut from Calcite main and it also returns 558, although the BigQuery docs have 559 as the answer so it appears unrelated to these changes, I'm looking more into it.